graphql/schema.graphql

```diff
@@ -484,3 +529,8 @@ enum Gender {
     FEMALE
     NA
 }
+
```

Comment:
Status is Compusarly or Optional.

Reply:
Everything I added is optional

Reply:
I mean the value should be Compusarly Subject or Optional Subject.

Reply:
What do Pending and Done represent? Don't think it's significant?

Reply:
@Nyakaru can you resolve this. we need to close this asap!

Reply:
Sure on it